routes/apiRoutes.js

```diff
@@ -43,3 +44,19 @@ module.exports = function (app) {
         res.redirect("/login");
     });
 };
+
+app.put("/api/gotchi/:id", function(req, res) {
+    if (!req.user) {
+        res.redirect("/login");
+    } else {
+        db.gotchi.update(
+            req.body,
+            {
+                where: {
+                    id: req.params.id
```

Comment:
I'm pretty sure this actually needs to be 
```
where: {
    UserId: req.params.id
}
```
because we're updating the Gotchi that belongs to that user.